cli/command/formatter/node.go

```diff
@@ -12,7 +14,7 @@ import (
 )
 
 const (
-	defaultNodeTableFormat           = "table {{.ID}} {{if .Self}}*{{else}} {{ end }}\t{{.Hostname}}\t{{.Status}}\t{{.Availability}}\t{{.ManagerStatus}}"
+	defaultNodeTableFormat           = "table {{.ID}} {{if .Self}}*{{else}} {{ end }}\t{{.Hostname}}\t{{.Status}}\t{{.Availability}}\t{{.ManagerStatus}}{{if .TLSStatus}}\t{{.TLSStatus}}{{end}}"
```

Comment:
Possibly - I'm not sure what other errors we should include, though.  There's already an availability column, and likely the ones that are failing to renew are the down or unavailable ones, but it's possible that they went down after renewing.

The other option is to not include the TLS status here, but to include a list of nodes that need certificate renewals in the CLI in #32993 - I was just worried this list would be too long if the user had like 1000 nodes.